onnx_xla/onnx_xla_interface.cc

```diff
@@ -55,26 +57,24 @@ struct BackendControl {
 //TODO: Determining # of CPU, GPU, TPU devices and return
 ONNXIFI_PUBLIC ONNXIFI_CHECK_RESULT onnxStatus ONNXIFI_ABI ONNXIFI_SYMBOL_NAME(
     onnxGetBackendIDs)(onnxBackendID *backendIDs, size_t *numBackends) {
-  try {
-    *backendIDs = (onnxBackendID)(new OnnxXlaBackendID());
+  ONNXIFI_TRY_CATCH(([&, backendIDs, numBackends] {
```

Comment:
If the function fails, `*numBackends` must be 0

Reply:
Good point - set it to 0 at beginning of try-catch